Sources/SwiftInspectorCore/TypeConformanceAnalyzer.swift

```diff
@@ -56,6 +62,24 @@ public final class TypeConformanceAnalyzer: Analyzer {
     let syntaxTypeName = String(describing: node.typeName).trimmingCharacters(in: .whitespaces)
     return (syntaxTypeName == self.typeName)
   }
+
+  private func findConformingType(of node: SyntaxProtocol?) -> String? {
+    guard let originalNode = node else { return nil }
+
+    guard let parent = originalNode.parent else { return nil }
+
+    // A conforming type can only be a class, struct or enum
+    // See: https://docs.swift.org/swift-book/LanguageGuide/Protocols.html
+    if let classSyntax = parent.as(ClassDeclSyntax.self) {
```

Comment:
asking to learn: where does the `func as` method come from?

Reply:
@dfed this is from SwiftSyntax, it was just introduced in the Swift 5.2 version, [see here for more](https://github.com/apple/swift-syntax/blob/master/Changelog.md#swift-52)